File: Programmers/Level1/2016년/2016년.py
# datetime 모듈 사용
import datetime
def solution(a, b):
    date = datetime.datetime(2016, a, b)
    return date.strftime("%a").upper())

# 일반적인 알고리즘 사용
def solution(a, b):
    months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    days = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    return days[(sum(months[:a - 1]) + b - 1) % 7]

File: Programmers/Level1/2016년/2016년.js
// Date 객체 사용
function solution(a, b) {
    return (answer = new Date(`2016 ${a} ${b}`)
        .toDateString()
        .split(" ")[0]
        .toUpperCase());
}

// 일반적인 알고리즘 사용
function solution(a, b) {
    const months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    const days = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"];
    let sum = 0;
    for (let i = 0; i < a - 1; i++) {
        sum += months[i];
    }

    return days[(sum + b - 1) % 7];
}
